css/talent/main.css

```diff
@@ -8,8 +8,63 @@ body {
   }
 }
 
+@font-face {
```

Comment:
Same here, it's a duplicate, if we put it in the css/main.css we don't need to duplicate, and we won't have project specific css interfering with page specific css.